auto_tests/onion_test.c

```diff
@@ -471,10 +472,24 @@ static Onions *new_onions(const Memory *mem, const Random *rng, uint16_t port, u
         return nullptr;
     }
 
+    on->tcp_np = netprof_new(on->log, mem);
```

Comment:
the order here is different to messenger, but i guess that just means more testing :)